### ISSUE TITLE
System freeze while attempting to install LMDE4 

### ISSUE THREAD

**Issue by elemmaiii** [linuxmint/lmde-4-cinnamon-beta#16]
Hello. Each time I attempt to install or even navigate in a live session with LMDE4 my computer freezes after a few minutes. I have an NVIDIA card, and I've attempted to boot with and without the NVIDIA session. My computer specs are as follows:

Machine:
  Type: Desktop System: Dell product: OptiPlex 7010 v: 01 
  serial: <root required> 
  Mobo: Dell model: 0GY6Y8 v: A02 serial: <root required> BIOS: Dell v: A28 
  date: 02/22/2018 
CPU:
  Quad Core: Intel Core i7-3770 type: MT MCP speed: 1597 MHz 
  min/max: 1600/3900 MHz 
Graphics:
  Device-1: NVIDIA GK106 [GeForce GTX 650 Ti Boost] driver: nvidia v: 435.21 
  Display: x11 server: X.Org 1.20.5 driver: nvidia 
  unloaded: fbdev,modesetting,nouveau,vesa 
  resolution: 1920x1200~60Hz, 1680x1050~60Hz 
  OpenGL: renderer: GeForce GTX 650 Ti BOOST/PCIe/SSE2 
  v: 4.6.0 NVIDIA 435.21 
Network:
  Device-1: Intel 82579LM Gigabit Network driver: e1000e 
  Device-2: Ralink RT5572 Wireless Adapter type: USB driver: rt2800usb

**Comment by Phil995511** [comment]
It run perfectly on my Dell XPS 9570 laptop with Nvidia 1050 graphic card ?

Test a install with LMDE 4 compatibility mode option.

**Comment by elemmaiii** [comment]
I'm able to commence install without freeze. However, the install (after using sudo live-installer) says that it's "Testing with random pattern:" and it's taking forever.

**Comment by clefebvre** [comment]
Maybe that's taking all your CPU... that's the part which erases the disk when you choose to use disk encryption.

**Comment by elemmaiii** [comment]
I gave up. I'm on Linux Mint Ubuntu. 
Between having to load the live environment in compatibility mode, then having to wait for hours to allow the install to take place, there's just not enough time in the day.
I still had to install Linux Mint Ubuntu in compatibility mode, but at least I was able to keep my computer up long enough to load the proprietary driver for my NVIDIA driver. That's more than I can say for LMDE4. It was just a losing proposition for me.
